CONTRIBUTING.md

```diff
@@ -0,0 +1,346 @@
+# Contributing to opentelemetry-go-build-tools
```

Comment:
Can you link to this file in `README.md`?

Reply:
Done